site.yml

```diff
@@ -74,9 +74,9 @@ asciidoc:
     latest-ocis-version: 'next'
     previous-ocis-version: 'next'
     # these versions are just for printing like in releases but not used for referencing
-    ocis-actual-version: '5.0.4'
+    ocis-actual-version: '5.0.8'
```

Comment:
This seems strange. Should we make a separate PR to update now from 5.0.4 to 5.0.7 ?
(and update the date at line 79)

Reply:
The site.yml file in this repo is for local building only.
I forgot in the past to update it. Non critical as the life value comes from docs/site.yml